### PR TITLE
Update change-address-contact-and-more.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the character limit as a note

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8916

File: microsoft-365/admin/manage/change-address-contact-and-more.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ To change other information on your company's profile page:
 2. On the **Organization profile** tab, select **Organization information**.
 3. Update your organization's information, then select **Save changes**. Be sure to fill in all required fields marked with an * to enable saving your changes.
 
+> [!NOTE]
+> Sharepoint online and Onedrive has a 256 character limit on your Windows PC. As users synchronize the document libraries to their PCs, they will encounter the error messages when trying to do anything within synchronized document libraries (i.e., create folders, rename documents, etc.).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/change-address-contact-and-more.md]:
```suggestion
> SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize the document libraries to your PC you will encounter error messages when attempting to do anything within the synchronized document libraries (such as creating folders or renaming documents).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder.

File: microsoft-365/admin/manage/change-address-contact-and-more.md [hunk 2]
```diff
@@ -52,6 +52,9 @@ To change other information on your company's profile page:
 2. On the **Organization profile** tab, select **Organization information**.
 3. Update your organization's information, then select **Save changes**. Be sure to fill in all required fields marked with an * to enable saving your changes.
 
+> [!NOTE]
+> SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize the document libraries to your PC you will encounter error messages when attempting to do anything within the synchronized document libraries (such as creating folders or renaming documents).
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/manage/change-address-contact-and-more.md]: Please replace paragraph with:

SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize document libraries to your PC, you will receive error messages when you attempt to do anything within the synchronized document libraries, such as creating folders or renaming documents.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please update the new paragraph as indicated.

**Comment by msbemba** [comment]
Thank you @cmcatee-MSFT  This has been updated.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please review and merge. Thanks

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Change 2nd sentence to:

If you exceed the character limit, you receive an error message when you try to do anything within the synchronized document libraries, like creating folders or renaming documents.

**Comment by yogkumgit** [comment]
@msbemba Please implement the suggested change. Thanks

**Comment by msbemba** [comment]
@cmcatee-MSFT I have updated the document as per your suggestion.

**Review by cmcatee-MSFT — APPROVED** [review]

**Pull request merged**